plugins/autoid/plugin.js

```diff
@@ -103,10 +103,12 @@
 
         for (i = 0; i < pastedElements.length; i++) {
           element = pastedElements[i];
-          id = element.attributes.id;
-          originalHeading = checkForDuplicateId(id);
-          if (originalHeading) {
-            element = resolveDuplicateIds(element, originalHeading);
+          if (element.attributes) {
```

Comment:
ah... I like your suggestion better.  A little more verbose, but I think it makes the intent much clearer